src/utils/dom.ts

```diff
@@ -1,13 +1,35 @@
-import { LiveSelector, MutationObserverWatcher } from '@holoflows/kit'
-import { timeout } from './utils'
+import { LiveSelector } from '@holoflows/kit'
+import { isUndefined } from 'lodash-es'
 
 export const untilElementAvailable = async (ls: LiveSelector<HTMLElement, boolean>) => {
-    await timeout(new MutationObserverWatcher(ls).startWatch({ childList: true, subtree: true }).then(), 10000)
+    return new Promise<void>((resolve, reject) => {
```

Comment:
why change to this way to impl?

Reply:
the prev implement was not working. maybe delay fixing?

Reply:
maybe leave an issue for it